API/JsonLd/HydraImageCollection.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace API.JsonLd
+{
+    public class HydraImageCollection : HydraCollectionBase
```

Comment:
Could this be `: SimpleCollection<Image>`

Reply:
It gets the PartialView from the hydraCollectionbase